### DISCUSSION TITLE
The Pinia Store may not be loaded

### DISCUSSION THREAD

**Discussion by timi137137** [vuejs/pinia#1546] [Help and Questions]
I have a lot of stores, but only two of them are properly registered. The others are working, but they don't seem to be properly registered, so I can't listen for events

This is what proper registration looks like
![image](https://user-images.githubusercontent.com/36812439/184472156-6a633796-7b2c-4147-817f-0ce9595fe9f5.png)

But I actually have these stores
![image](https://user-images.githubusercontent.com/36812439/184472280-98a99ccd-9166-48b5-9cb1-1c69e053f08f.png)

in main.ts
![image](https://user-images.githubusercontent.com/36812439/184472298-2bfbb69e-0706-4103-9798-41ba5704280d.png)

I have been inquiring all morning, but I have no clue. Hoping for help

**Comment by posva** [comment]
Stores are loaded when their `use...Store()` function gets called for the first time

**Reply by timi137137** [reply]
![image](https://user-images.githubusercontent.com/36812439/184884930-e5a403c6-b69c-4017-bbd1-389f9d938883.png)
I made sure that I called the method and that every route was called before it was loaded
but The problem remains

**Reply by posva** [reply]
They must be called after `app.use(pinia)`: https://pinia.vuejs.org/core-concepts/#using-the-store

Maybe the devtools manage to load after these calls. The message you see is just devtools

**Reply by timi137137** [reply]
I'm asking this question because I found that Store.$subscribe doesn't work for stores that don't show loading, so my persistence doesn't work either

I'm sure they work, but stores without a DevTools prompt can't use this event

**Reply by timi137137** [reply]
I have tried to use async, change the sequence of code and other methods to eliminate the error, but it still exists, there is no way to help
